### PR TITLE
test: fix some bugs for test

### PR DESCRIPTION
**Pull request by rudyfly**
### Ⅰ. Describe what this PR did
add start lxcfs
get the root dir of pouch daemon, instead of `/var/lib/pouch`
modify ipv6 test for alikernel

### Ⅱ. Does this pull request fix one issue?

NONE

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
Just modify test

### Ⅴ. Special notes for reviews

Signed-off-by: Rudy Zhang <rudyflyzhang@gmail.com>

### CODE REVIEW

File: test/cli_inspect_test.go [hunk 1]
```diff
@@ -36,6 +36,26 @@ func (suite *PouchInspectSuite) TearDownTest(c *check.C) {
 // TestInspectCreateAndStartedFormat is to verify the format flag of inspect command.
 func (suite *PouchInspectSuite) TestInspectCreateAndStartedFormat(c *check.C) {
 	name := "TestInspectCreateAndStartedFormat"
+	// get root dir
+	ret := command.PouchRun("info")
```
[the remark this answers is not in the document]
Comment (rudyfly) [reply test/cli_inspect_test.go]: get it

**Comment by HusterWan** [comment]
please rebase your code

File: test/cli_inspect_test.go [hunk 2]
```diff
@@ -68,7 +73,7 @@ func (suite *PouchInspectSuite) TestInspectCreateAndStartedFormat(c *check.C) {
 	// Inspect LogPath, LogPath should not be empty after container's start.
 	// by default, the container has log type of json-file.
 	output = command.PouchRun("inspect", "-f", "{{.LogPath}}", name).Stdout()
-	expectedLogPath := fmt.Sprintf("/var/lib/pouch/containers/%s/json.log", containerID)
```
Comment (ZYecho) [inline test/cli_inspect_test.go]: I search in pouch , find more hard code `/var/lib/pouch`, maybe we can fix in this pr to replace it use this func?

Comment (rudyfly) [reply test/cli_inspect_test.go]: I can't find anyone in testcase....Can you give a case?

Comment (ZYecho) [reply test/cli_inspect_test.go]: I mean in whole project can do as the same way, not only the testcase.

Comment (zhuangqh) [reply test/cli_inspect_test.go]: I've searched the whole project. Actually, `/var/lib/pouch` occurred in 5 go file. 
One of them is the default config of pouchd. Two of them is the default config of storage.

Two of them inside the test.
First one, a mock test, acceptable.
https://github.com/alibaba/pouch/blob/1f7dbf60d406210c65fb96ea4682f6d04f97773e/client/system_info_test.go#L40

Second one, a hard code path in cli test.
https://github.com/alibaba/pouch/blob/638b38fa9a09ae52922a900d611817b69e017d7c/test/cli_inspect_test.go#L71
I think this one could use `GetRootDir` function for more generality.

@ZYecho @rudyfly

Comment (ZYecho) [reply test/cli_inspect_test.go]: LGTM @zhuangqh thanks for your check.

**Comment by zhuangqh** [comment]
LGTM

**Pull request merged**